### ISSUE TITLE
create fails at: Installation request for symfony/config v3.4.20 -> satisfiable by symfony/config[v3.4.20]

### ISSUE THREAD

**Issue by nagyv** [matzeeable/create-wp-react-app#2]
Hi,

I've tried to run create-wp-react-app in a "Windows Subsystem for Linux", but it fails with symphony:

```shell
$ npx create-wp-react-app create wsb-players
npx: installed 105 in 23.048s
  ✔ General checks for wsb-players... Passed! Passed!
  ❯ Download boilerplate
    ✔ Clone git repository
    ✔ Disconnect git repository
    ✔ Install package dependencies with npm
    ✖ Install package dependencies with composer
      →     - Installation request for symfony/config v3.4.20 -> satisfiable by symfony/config[v3.4.20].
```

Is there anything I can do to debug this issue and get it working?

My environment is:

```shell
$ composer -V
Composer version 1.8.4 2019-02-11 10:52:10
$ php -v
PHP 7.0.33-0ubuntu0.16.04.2 (cli) ( NTS )
Copyright (c) 1997-2017 The PHP Group
Zend Engine v3.0.0, Copyright (c) 1998-2017 Zend Technologies
    with Zend OPcache v7.0.33-0ubuntu0.16.04.2, Copyright (c) 1999-2017, by Zend Technologies
$ node -v
v8.15.1
$ npm -v
6.4.1
```

**Comment by matzeeable** [comment]
Hi! Sorry for my late reply. Did you get it to work with windows?

**Comment by nagyv** [comment]
Yes, I've forked your repo.

Unfortunately, I don't know a platform-neutral way to run `npm run phpdocs` as vendor/bin is not in path. For this reason, I did not open a PR, otherwise my fork is working fine.

See the last 3 commits:

1. https://github.com/nagyv/wp-reactjs-starter/commit/a5679efab4f2e77787789e69e00e68c9a52673b6
1. https://github.com/nagyv/wp-reactjs-starter/commit/1c0500673bf42dfbc2535040afb385f3ce065cc4
1. https://github.com/nagyv/wp-reactjs-starter/commit/40d3f2285168481286ef64f15b494766164a1f20
